tests/ape-config.yaml

```diff
@@ -9,8 +9,10 @@ hardhat:
     ethereum:
       mainnet:
         upstream_provider: alchemy
+        block_number: 15000000
```

Comment:
Specifying block number allows cache to get created which speeds things up and saves API requests:
 https://hardhat.org/hardhat-network/docs/guides/forking-other-networks#pinning-a-block